cmd.go

```diff
@@ -672,6 +678,10 @@ func passToMakepkgCapture(dir string, args ...string) (string, string, error) {
 	mflags := strings.Fields(config.MFlags)
 	args = append(args, mflags...)
 
+	if config.MakepkgConf != "" {
```

Comment:
I know we leave a lot of things to user responsibility but here would be an easy spot to check if file exists and:
1. Warn user file does not exist
2. Don't pass --config if file does not exist